README.md

```diff
@@ -75,6 +80,6 @@ Our [wiki](https://github.com/CorsixTH/CorsixTH/wiki) is a good place to start,
 ## Contact
 
 - Follow us on [Reddit](https://www.reddit.com/r/corsixth), Twitter ([**@CorsixTH**](https://twitter.com/CorsixTH)), and on [Facebook](https://facebook.com/CorsixTH)
-- IRC [#Corsix-TH](https://webchat.freenode.net/#corsix-th) on irc.freenode.net
+- Hit up our [Matrix Room (Discord Bridge)](https://matrix.to/#/#corsixth-general:matrix.org) (web acccess available via [Element.io](https://element.io/get-started))
```

Comment:
the web access link is probably not needed, iirc the matrix.to link is pretty informative.

Reply:
```suggestion
- Hit up our [Matrix Room (Discord Bridge)](https://matrix.to/#/#corsixth-general:matrix.org)
```